poetry.lock

```diff
@@ -649,7 +649,7 @@ bcrypt = ["bcrypt"]
 
 [[package]]
 name = "django-ansible-base"
-version = "2025.1.31"
+version = "2025.1.3"
```

Comment:
This is an older ref of DAB. I believe this is wrong or maybe a ref development that now in main it should point to a new release.